packages/sdk/odahuflow/sdk/clients/api.py

```diff
@@ -384,6 +386,16 @@ def __init__(self,
         self.authenticator = Authenticator(client_id, client_secret, non_interactive, base_url, token)
         self.conn_policy = HTTPConnectionPolicy(retries, timeout)
 
+        retry_strategy = Retry(
+            total=self.conn_policy.get_left_retries(),
```

Comment:
Maybe we should refactor/remove Connection Policy class align-with this diff? Seems like its functionality now is limited to 2 getter methods - for timeout and for retries. And moreover, `get_left_retries()` is used right next to an actual object construction. So it could be just `total=retries`.